vignettes/scoringutils.Rmd

```diff
@@ -19,6 +19,7 @@ library(magrittr)
 library(data.table)
 library(ggplot2)
 library(knitr)
+data.table::setDTthreads(2) # restricts number of cores used on CRAN
```

Comment:
do you really want this comment about CRAN here? Suggest change to something more useful to the user (i.e. you can change this to better fit the requirements of your machine.

Reply:
I updated the comment (although users would only see it if they look at the vignette .Rmd file)